src/sync/lookupApiUtils.js

```diff
@@ -37,20 +45,22 @@ export const createPolicyRecord = policy => {
   UIDatabase.write(() => createRecord(UIDatabase, 'InsurancePolicy', policyRecord));
 };
 
-const getQueryString = params =>
-  params.reduce((queryString, param) => {
-    const [[key, value]] = Object.entries(param);
-    if (!value) return queryString;
-    const paramString = `${key}=${SEPARATORS.QUERY_WILDCARD}${value}${SEPARATORS.QUERY_WILDCARD}`;
-    const paramSeparator = queryString.length > 0 ? SEPARATORS.PARAMETERS : SEPARATORS.QUERY_STRING;
-    return queryString + paramSeparator + paramString;
-  }, '');
+const getQueryString = params => {
+  const object = params.reduce((queryObject, param) => {
+    const [[key, value], [, type]] = Object.entries(param);
+    if (!value) return queryObject;
+    const paramValue = type !== TYPES.DATE ? `@${value}@` : moment(value).format('YYYYMMDD');
+    return { ...queryObject, [key]: paramValue };
+  }, {});
+  const queryString = `?${querystring.stringify(object)}`;
+  return queryString;
```

Comment:
mainly because the mix of 'querystring' and 'queryString' tripped me up.
```suggestion
  return `?${querystring.stringify(object)}`;
```

Reply:
49ca9cc.